iceoryx_posh/test/moduletests/test_roudi_portmanager.cpp

```diff
@@ -207,10 +207,10 @@ void setDestroyFlagAndClearContainer(vector& container)
     container.clear();
 }
 
-TEST_F(PortManager_test, doDiscovery_singleShotPublisherFirst)
+TEST_F(PortManager_test, DoDiscoveryWithSingleShotPublisherFirst)
 {
-    PublisherOptions publisherOptions{1, "node"};
-    SubscriberOptions subscriberOptions{1, 1, "node"};
+    PublisherOptions publisherOptions{1, "node", false};
```

Comment:
I guess @dkroenke would say: 'the big U is your friend' ;)

Reply:
OK, this one is dedicated to @dkroenke